dev-tools/src/main/openshift/kapua-template.yml

```diff
@@ -21,7 +21,7 @@ parameters:
   value:            jolokia
 - name:           ELASTIC_SEARCH_MEMORY
   descrption:       The minimum and maximun Java heap space for Elasticsearch
-  value:            512m
+  value:            "-Xms512m -Xmx512m"
```

Comment:
This parameter really should be the memory, not the Xms/Xmx settings, please transform this to the proper JVM settings below: `-Xms${…} -Xmx${…}`